tests/capture/test_capture_qnode.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for capturing a qnode into jaxpr.
+"""
+from functools import partial
+
+# pylint: disable=protected-access
+import pytest
+
+import pennylane as qml
+from pennylane.capture.capture_qnode import _get_qnode_prim
+
+qnode_prim = _get_qnode_prim()
+
+pytestmark = pytest.mark.jax
+
+jax = pytest.importorskip("jax")
+
+
+@pytest.fixture(autouse=True)
+def enable_disable_plxpr():
+    qml.capture.enable()
+    yield
+    qml.capture.disable()
```

Comment:
Could start thinking about putting this into a `tests/capture/conftest.py` :sweat_smile:

Reply:
The challenge is that we wouldn't want it to run with `test_switches.py` 🤔

Reply:
Could do something like [this](https://stackoverflow.com/a/38763328/13489115)

Reply:
Good by me to resolve this comment.